modules/terraform/format.go

```diff
@@ -23,6 +24,20 @@ func FormatTerraformVarsAsArgs(vars map[string]interface{}) []string {
 	return formatTerraformArgs(vars, "-var")
 }
 
+// FormatTerraformLockAsArgs formats the lock and lock-timeout variables
+// -lock, -lock-timeout
+func FormatTerraformLockAsArgs(lockCheck bool, lockTimeout string) []string {
+	var lockArgs []string
+	if lockCheck == true {
```

Comment:
```suggestion
	if lockCheck {
```

Reply:
Also, if this is `false`... Shouldn't we set `-lock=false`?

Reply:
Good point, thanks for looking over!
I've updated the code so that it adds a -lock=false if lock is set to false and changed the conditional to the preferred syntax